front/scripts/main/models/DiscussionPostModel.ts

```
@@ -14,6 +14,7 @@ App.DiscussionPostModel = Em.Object.extend(App.DiscussionErrorMixin, {
 	page: 0,
 	connectionError: null,
 	notFoundError: null,
+	minorError: null,
```

It would be nice to add a comment explaining what do we need it for and in what cases we want to fail silently.